frontend/src/styles/home/FeaturedProjectCard.module.css

```diff
@@ -1,33 +1,19 @@
-
 #cardContainer {
-  margin-top: 0px;
-  margin-bottom: 3vw;
-  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 15vw;
 }
 
 #cardImageContainer {
-  flex: 0.35;
   border-color: #333333;
   justify-content: center;
   align-items: center;
   overflow: hidden;
-  display: flex;
-  width: 300px;
+  width: auto;
   height: 200px;
   border-radius: 50px;
-}
-
-#cardTextContainer {
-  flex: 0.65;
-  left: 0;
-  top: 0;
-  margin-left: 2vw;
-  max-height: 200px;
-  height: 200px;
-}
-
-#cardHeadingContainer {
-  display: flex;
+  margin-bottom: 7vw;
 }
 
 #cardImage {

```

Comment:
does this actually do anything

Reply:
@heylookitsme This was to create some whitespace between the image and the text when in mobile view as shown in the Figma. Without this, the Title text would be hugging the bottom edge of the pictures.
<img width="250" alt="Screenshot" src="https://user-images.githubusercontent.com/52669318/183266449-374eab79-2be3-49de-a387-45564297e46d.png">